specs/policy/attributes/attributes.openapi.yaml

```diff
@@ -137,9 +137,11 @@ paths:
                Fully Qualified Names of attribute values (i.e. https://<namespace>/attr/<attribute_name>/value/<value_name>), normalized to lower case.
         - name: withValue.withKeyAccessGrants
           in: query
+          description: Deprecated
           schema:
             type: boolean
             title: with_key_access_grants
+            description: Deprecated
```

Comment:
Do you want to include the recommended approach for this as well (like you did other places)?

Reply:
See https://github.com/opentdf/docs/pull/143#discussion_r2298665001